### PR TITLE
Create 2018-11-08-a-new-home-for-tutorials.md

### PR DESCRIPTION
**Pull request by locks**
Announcement for https://tutorials.emberjs.com.

### CODE REVIEW

File: source/blog/2018-11-08-a-new-home-for-tutorials.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: A new home for Tutorials
+author: Learning Core Team
+tags: Recent Posts
+responsive: true
+---
+
+TBK
```
Comment (acorncom) [inline source/blog/2018-11-08-a-new-home-for-tutorials.md]: I like the idea, but the implementation seems ... _lacking_? 😉

Comment (MelSumner) [reply source/blog/2018-11-08-a-new-home-for-tutorials.md]: @locks what do you want the tutorial announcement to be?

Comment (locks) [reply source/blog/2018-11-08-a-new-home-for-tutorials.md]: I want to explain the decision of moving the tutorials out of the guides and making them non-versioned, and give a tiny glimpse of our roadmap if the team agrees.

**Comment by MelSumner** [comment]
Closing this because it never left WIP status and we've moved to a new repository.

**Pull request closed without merging**